ansible/roles/remove_local_storage/templates/localvolume.yml.j2

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: "local.storage.openshift.io/v1"
```

Comment:
Since this and roles/local_storage/templates/localvolume.yml.j2  are same can we symlink?